cocotb/runner.py

```diff
@@ -856,15 +858,17 @@ def build_command(self) -> List[Command]:
             # + ["-plidebug"]  # Enhance the profile output with PLI info
             # + ["-plierr_verbose"]  # Expand handle info in PLI/VPI/VHPI messages
             # + ["-vpicompat 1800v2005"]  # <1364v1995|1364v2001|1364v2005|1800v2005> Specify the IEEE VPI
-            + ["-access +rwc"]
+            + ["-access"]
+            + ["+rwc"]
+            + ["-loadvpi"]
             + [
-                "-loadvpi "
-                + cocotb.config.lib_name_path("vpi", "xcelium")
+                cocotb.config.lib_name_path("vpi", "xcelium")
                 + ":vlog_startup_routines_bootstrap"
             ]
             + [f"-work {self.library_name}"]
             + self.compile_args
-            + ["-define COCOTB_SIM"]
+            + ["-define"]
+            + ["COCOTB_SIM"]
             + self.get_define_options(self.defines)
             + self.get_include_options(self.includes)
             + self.get_parameter_options(self.parameters)
```

Comment:
Looks to me like these three functions are also wrong.

Reply:
Sorry @eric-wieser, I didn't see your comment before merging. I opened https://github.com/cocotb/cocotb/issues/3134 to track, @tianrui-wei if you have time, would you mind having a look?

Reply:
Hi @imphil , @eric-wieser ,

Thanks for pointing them out. I'll conduct some more testing and fix the issue should I find one.

Best,
Tianrui